lib/active_merchant/billing/gateways/pay_arc.rb

```diff
@@ -296,7 +296,7 @@ def add_address(post, creditcard, options)
 
       def add_money(post, money, options)
         post['amount'] = money
-        post['currency'] = (options[:currency] || currency(money))
+        post['currency'] = currency(money) unless options[:currency]
```

Comment:
So let me see if I have this right: if there is a value in `options[:currency]` we don't need to explicitly add it here, since it will eventually get added in `filter_gateway_fields` with `post.update(filtered_options)`?

Reply:
Yep!